biolearn/data/library.yaml

```diff
@@ -864,4 +864,283 @@ items:
     path: custom
     parser:
       type: jen-age-custom
-      data-type: rna
\ No newline at end of file
+      data-type: rna
+  - id: GSE167998
+    title: "FlowSorted.BloodExtended.EPIC: An Enhanced DNA Methylation Library for
+      Deconvoluting Peripheral Blood"
+    summary: DNA methylation microarrays have been extensively used for
+      understanding cell type composition in complex tissue samples. Here we
+      expand on existing libraries for reference-based deconvolution of blood DNA
+      methylation data assayed using the Illumina HumanMethylationEPIC array to
+      include 12 different leukocyte subtypes (neutrophils, eosinophils,
+      basophils, monocytes, B cells naïve and memory, CD4+ and CD8+ naïve and
+      memory cells, natural killers, and T regulatory cells). Application of the
+      IDOL algorithm for identifying optimal libraries for the
+      deconvolution of blood-derived DNA methylation data led to to a library
+      consisting of 1200 CpGs. The accuracy of deconvolution estimates obtained
+      using our IDOL-optimized library was evaluated using artificial mixtures
+      with known cellular composition and in samples were both whole-blood DNA
+      methylation data and FACS information were available. We further
+      showcase potential applications of our expanded library using
+      publicly available cancer, aging, and autoimmune disease data sets.
+    format: Unknown
```

Comment:
It looks like the script failed to determine the format automatically. It's probably IlluminaEPIC but we should verify. Same with other unknown formats. You can check the platform on the GEO site to confirm https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167998

Reply:
Yes these three datasets have the IlluminaMethylationEPIC format.

Reply:
Sanity check on zip file [Test.zip](https://github.com/user-attachments/files/16566050/Test.zip)
Please let me know if you are able to review the tests or if you want me to send the sanity checks in another format.